contrib/google.golang.org/grpc/client_stats_handler.go

```diff
@@ -0,0 +1,62 @@
+package grpc
```

Comment:
Do you mind shortening the names of the files by removing the `_handler` suffix from all of them? Unless you have a good reason go keep them. You might also get nicer grouping in the file list by using:
```go
stats_client.go
stats_server.go
```
This way they will be grouped together. But I don't feel strongly about this. Point is to shorten a bit if possible.